### ISSUE TITLE
Cleanup IrsPlan module

### ISSUE THREAD

**Issue by moshthepitt** [onaio/reveal-frontend#416]
After #402, the `IrsPlan` module and component needs to be audited for unused code and then cleaned.

**Comment by cKellyDesign** [comment]
- [ ] Removes New Plan functionality
- [ ] Removes Edit Plan Name functionality
- [ ] Adds missing docstrings
- [ ] Removes isNewPlan state / functionality
- [ ] Removes unused imports and variables
